src/hooks/useHover.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+// Hooks
+const useHover = () => {
+  const [hovered, setHovered] = useState();
+  const [timer, setTimer] = useState();
+
+  const eventHandlers = {
+    onMouseOver() {
+      setTimer(setTimeout(() => setHovered(true), 700));
```

Comment:
Maybe we can make this 700 config too. But I don't know if it is worth.~

Reply:
haha, premature optimization is the root of all evil. Currently, we should modularize all components and find out which part of logic is independent or dependent, and abstract them respectively.